neonUtilities/R/byTileAOP.R

```diff
@@ -43,7 +43,17 @@ byTileAOP <- function(dpID, site="SJER", year="2017", easting, northing, buffer=
   if(substring(dpID, 3, 3)!=3 & dpID!="DP1.30003.001") {
     stop(paste(dpID, "is not a Level 3 data product ID.\nThis function will only work correctly on mosaicked data.", sep=" "))
   }
-  
+
+  # error message if site is left blank
+  if(regexpr('[[:alpha:]]{4}', site)!=1) {
```

Comment:
Suggested rephrase: 'A four-letter site code is required. NEON sites codes can be found here: <link>'